src/javascript/tracking.js

```diff
@@ -193,10 +193,10 @@ const tracking = {
 	destroy,
 	toString,
 	init,
-	click: initClick,
+	click,
```

Comment:
Would it be too odd to make this `click: { initClick }` or we could add a named export in the `click.js` file which returns an object?

Reply:
I think `click.js` already returns a named exports `export {
	init
};`

Reply:
`init` in this case isn't an object, but the export of one item. If we were to make it a default then it would be an object, so could change this back to `export default { init };` and that would work with the other change -> `import click from './events/click.js';`

Or as a named export, maybe this would work, but it's not too clear

```js
const initClick = { init };
export { initClick as init };
```

Reply:
ah my bad, I read that as a default export :)

Reply:
I've updated this to be more like `/events/page-view.js`. An object is returned with an `init` method